tests/test_admin.py

```diff
@@ -116,3 +116,54 @@ async def test_resurrect_switch(setup_tables_and_data, client, login, switch):
     content = await response.content.read()
 
     assert 'switch3' in content.decode('utf-8')
+
+
+async def test_switches_copy_without_auhtorize(setup_tables_and_data, client):
+    response = await client.post('/zbs/switches/copy')
+
+    assert response.status == 401
+
+
+@pytest.mark.usefixtures('setup_tables_and_data', 'get_switches_data_mocked_existing_switch')
+async def test_switches_copy_existing_switch(client, login):
+    response = await client.post('/zbs/switches/copy')
+    async with client.server.app['db'].acquire() as conn:
```

Comment:
This piece of code is repeated here and in the tests below. Let's move it to the helper.

Reply:
Fixed in: https://github.com/best-doctor/its_on/pull/15/commits/7bdd33d84fc2d68073653df9c36b9858c20cedec